docs/pages/setup/reference/cli.mdx

```diff
@@ -1305,6 +1305,360 @@ $ tctl auth rotate --type=user --grace-period=200h
 $ tctl auth rotate --type=host --grace-period=8h
 ```
 
+### tctl sso test
+
+Perform end-to-end test of SSO flow using provided auth connector definition.
+
+The command supports all auth connector types: `github`, `oidc`, `saml`. The latter two require Teleport Enterprise.
+
+The testing consists of running a single end-to-end authentication request against provided auth connector definition.
+Once the request is finished, the results will be printed to console along with context-specific diagnostic informations.
+The test process will modify the list of configured auth connectors or result in certificates being issued.
+
+```code
+$ tctl [<global-flags>] sso test [<filename>]
+```
+
+<Admonition
+  type="note"
+  title="Important"
+>
+  To use this command, you must have access to `github_request`, `oidc_request`, `saml_request` resources (depending on the type of connector being tested).
+
+  If you receive "permission denied" error, modify one of your roles to allow access to:
```

Comment:
Instead of suggesting that a user directly modifies their role (which is probably not possible in many enterprise environments), let's just say "ensure that one of your roles allows access to"